bash_completion

```diff
@@ -30,58 +30,58 @@ __nvm_commands ()
     cache deactivate unload
     version version-remote which'
 
-    if [ ${#COMP_WORDS[@]} == 4 ]; then
+  if [ ${#COMP_WORDS[@]} == 4 ]; then
 
-      command="${COMP_WORDS[COMP_CWORD-2]}"
-      case "${command}" in
+    command="${COMP_WORDS[COMP_CWORD - 2]}"
+    case "${command}" in
       alias)  __nvm_installed_nodes ;;
-      esac
+    esac
 
-    else
+  else
 
-      case "${current_word}" in
-      -*)     __nvm_options ;;
-      *)      __nvm_generate_completion "$COMMANDS" ;;
-      esac
+    case "${current_word}" in
+      -*)  __nvm_options ;;
+      *)   __nvm_generate_completion "${COMMANDS}" ;;
```

Comment:
these really only need one space after `)`

Reply:
Cool, I prefer that.